scripts/digest_inputs.bats

```diff
@@ -300,7 +308,7 @@ assert_failure_with_message_when() { local MESSAGE="$1"; shift
 
 @test "redhat_tag contains whitespace / error" {
 	set_all_required_env_vars_and_tags
-	export REDHAT_TAG="scan.connect.redhat.com/some/image:1.2.3 scan.connect.redhat.com/blah/blah:1.2.3"
+	export REDHAT_TAG="quay.io/redhat-isv-containers/blah:1.2.3"
```

Comment:
This seems to match the expected pattern unless I am missing something?

Reply:
I think the original test was to make sure there was only exactly 1 entry in `redhat_tag` right?

Reply:
Ah, yes, good spot thanks!